### DISCUSSION TITLE
Activities with hourly rates and tags?

### DISCUSSION THREAD

**Discussion by samuelT2** [kimai/kimai#3670] [Questions & Answers] [answered]
Hi all

I have a newly installed kimai instance, the tool is fantastic! Whereas I was able to figure out most of the functionality by clicking around, I have a question to the community left.

A project which we work on defines two rates: Amount X for support and amount Y for development. Each activity is a github issue and is labeled as support or development. Each time entry must refer to such an activity.

Perhaps my use case is wrongly adapted - but wouldn't it be handy to have a tag and hourly rate field within activities? Or is that something which I can achieve with the custom fields plugin? Or did I miss something in the settings?

Thank you very much for your help.

**Comment by kevinpapst** [comment, marked as answer]
You can set rates for customers, projects, activities and users. RTM.

Auto tagging is a nice feature and will come with the next major upgrade #2902 through the MetaFields bundle.

**Comment by samuelT2** [comment]
Thank you @kevinpapst - even though I had a look on the docu first, I was too dumb to realize that I need to click on the activity, not edit the activity.
